pkg/proxy/register.go

```diff
@@ -59,14 +60,13 @@ func (p *Register) Add(definition *RouterDefinition) error {
 
 	handler := NewBalancedReverseProxy(definition.Definition, balancerInstance, p.statsClient)
 	handler.FlushInterval = p.flushInterval
-	handler.Transport = &ochttp.Transport{
-		Base: transport.New(
-			transport.WithIdleConnTimeout(p.idleConnTimeout),
-			transport.WithInsecureSkipVerify(definition.InsecureSkipVerify),
-			transport.WithDialTimeout(time.Duration(definition.ForwardingTimeouts.DialTimeout)),
-			transport.WithResponseHeaderTimeout(time.Duration(definition.ForwardingTimeouts.ResponseHeaderTimeout)),
-		),
-	}
+	handler.Transport = transport.New(
```

Comment:
with this change we're losing transport metrics as I see

Reply:
The branch is still work in progress; we should still wrap transport with `ochttp`.

Reply:
Metrics are now back 🙌